Lombiq.ChartJs.Samples/Controllers/SampleController.cs

```diff
@@ -184,7 +184,10 @@ public async Task<IActionResult> History(string incomeTag = null, string expense
                         Date = dateIndex.Date!.Value,
                         Amount = numericIndex.Numeric ?? 0,
                     })
+            // Adding DateTimeKind requires an overload that needs 3 more parameters which seems to be overkill for our purposes.
+#pragma warning disable S6562 // Always set the "DateTimeKind" when creating new "DateTime" instances
             .GroupBy(transaction => new DateTime(transaction.Date.Year, transaction.Date.Month, 1))
+#pragma warning restore S6562 // Always set the "DateTimeKind" when creating new "DateTime" instances
```

Comment:
It's just 3 additional zeroes, right? Seems easier to just write it in than what you are doing here.